src/BackpackServiceProvider.php

```diff
@@ -20,7 +20,9 @@ class BackpackServiceProvider extends ServiceProvider
         \Backpack\CRUD\app\Console\Commands\CreateUser::class,
         \Backpack\CRUD\app\Console\Commands\PublishBackpackMiddleware::class,
         \Backpack\CRUD\app\Console\Commands\PublishView::class,
-        \Backpack\CRUD\app\Console\Commands\RequireDevTools::class,
+        \Backpack\CRUD\app\Console\Commands\Addons\RequireDevTools::class,
```

Comment:
My biggest problem with this PR - even though everything works fine, I keep getting this weird error upon each command execution. Previously (on `main`) the output was nice and clean:
<img width="1061" alt="CleanShot 2022-07-31 at 08 45 11@2x" src="https://user-images.githubusercontent.com/1032474/182012138-d7c95d21-c387-47b0-80c9-a5089c34d3ff.png">

But in this PR, I get `sh: mode: command not found` all the freakin' time:

<img width="1128" alt="CleanShot 2022-07-31 at 08 48 25@2x" src="https://user-images.githubusercontent.com/1032474/182012167-8348bab2-cd4b-4e4f-9f86-b9b90ee0ef30.png">

Any idea how to solve this @promatik ?

Reply:
This should now be fixed 👌
I took a look at termwind code 🤫 to see how they get the the console size 👌